<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LSCEpisodePreparation</class>
 <widget class="QWidget" name="LSCEpisodePreparation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSCEpisodePreparation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionExample_action">
   <property name="text">
    <string>Example action</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>

